app/views/slides/templates/_standard.html.erb

```diff
@@ -9,15 +9,21 @@
 </div>
 
 <% if slide.background_type == 'video' %>
-  <video id="js-background-video" src="<%= slide.background %>" class="ui-slide-background" preload='auto' loop></video>
+<%# `controls muted` attribute mutes videos automatically
+    Source: https://www.w3schools.com/tags/att_video_muted.asp %>
+  <!-- <video controls muted id="js-background-video" src="<%= slide.background %>" class="ui-slide-background" preload='auto' loop></video> -->
+
+  <%= video_background %>
 <% end %>
 
 <% if slide.background_type == 'image' %>
   <div class="ui-slide-background" style="width: 100%; height: 100%; background:url('<%= slide.background_url %>') no-repeat center; background-size: cover;"></div>
 <% end %>
 
 <% if slide.foreground_type == 'video' %>
-  <video id="js-foreground-video" src="<%= slide.foreground %>" class="ui-slide-foreground" loop style="width: 100%; height: 100%;"></video>
+<%# `controls muted` attribute mutes videos automatically
+    Source: https://www.w3schools.com/tags/att_video_muted.asp %>
+  <video controls muted id="js-foreground-video" src="<%= slide.foreground %>" class="ui-slide-foreground" loop style="width: 100%; height: 100%;"></video>
```

Comment:
And then this could be:

    <%= slide_video_tag(slide, :foreground) =>

In fact, it looks like whether it is a background or foreground slide can be derived from the `slide` record. So this could very well work for both cases:

    <%= slide_video_tag(slide) =>

Code comment here could not that this is a helper (but not absolutely necessary -- a developer could be expected to reasonably deduce that.)

Existing code comment could be moved to the helper.